--- a/CMakeLists.txt	Sun Nov 16 00:38:55 2014
+++ b/CMakeLists.txt	Sun Nov 16 00:37:51 2014
@@ -362,8 +362,10 @@
 set(HAVE_LIBSSL OFF)
 
 if(CMAKE_USE_OPENSSL)
-  find_package(OpenSSL)
-  if(OPENSSL_FOUND)
+  # FindOpenSSL.cmake is buggy under cross compilation and will
+  # fail no matter what you pass as OPENSSL_ROOT_DIR etc.
+  #find_package(OpenSSL)
+  #if(OPENSSL_FOUND)
     list(APPEND CURL_LIBS ${OPENSSL_LIBRARIES})
     set(USE_SSLEAY ON)
     set(USE_OPENSSL ON)
@@ -380,7 +382,7 @@
     check_include_file_concat("openssl/ssl.h"    HAVE_OPENSSL_SSL_H)
     check_include_file_concat("openssl/x509.h"   HAVE_OPENSSL_X509_H)
     check_include_file_concat("openssl/rand.h"   HAVE_OPENSSL_RAND_H)
-  endif(OPENSSL_FOUND)
+  #endif(OPENSSL_FOUND)
 endif(CMAKE_USE_OPENSSL)
 
 #libSSH2
